web-console/src/lib/functions/felderaRelation.ts

```diff
@@ -0,0 +1,28 @@
+import { Field, Relation } from '$lib/services/manager'
+import invariant from 'tiny-invariant'
+
+export type CaseDependentName = {
+  case_sensitive?: boolean | undefined
+  name: string
+}
+
+export const quotifyRelationName = (relation: Relation) =>
+  relation.case_sensitive ? `"${relation.name}"` : relation.name
+
+export const caseDependentName = (quotedRelationName: string) => {
+  const name = /\"?(\w+)\"?/.exec(quotedRelationName)?.[1]
+  invariant(name, `Invalid relation name parsed: ${name}`)
+  return {
+    name,
+    case_sensitive: /\"/.test(quotedRelationName)
+  }
+}
+
+export const caseDependentNameEq = (a: CaseDependentName) => (b: CaseDependentName) =>
```

Comment:
I really like methods, why don't you make these methods of the CaseDependentName type?
This way you pollute the global namespace with lots of functions.

Reply:
The UI is developed in functional programming style where result code with TypeScript syntax is reasonable. Having functions declared in a global namespace does not degrade development experience

Reply:
Neither does having methods